docs/developer-guide/nodejs-api.md

```diff
@@ -7,19 +7,19 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
 ## Table of Contents
 
 * [ESLint]
-    * [constructor()][eslint-constructor]
-    * [lintFiles()][eslint-lintFiles]
-    * [lintText()][eslint-lintText]
-    * [calculateConfigForFile()][eslint-calculateConfigForFile]
-    * [isPathIgnored()][eslint-isPathIgnored]
-    * [loadFormatter()][eslint-loadFormatter]
-    * [static version][eslint-version]
-    * [static outputFixes()][eslint-outputFixes]
-    * [static getErrorResults()][eslint-getErrorResults]
-    * [LintResult type](lintresult)
-    * [LintMessage type](lintmessage)
-    * [EditInfo type](editinfo)
-    * [Formatter type](formatter)
```

Comment:
There are definitions of `[eslint-constructor]`-like links at the end of this file. Would you fix those instead? Those links are used in other places as well.